app/controllers/api/v1/api_controller.rb

```diff
@@ -32,6 +32,16 @@ def log_exception(exception)
         logger.error exception.backtrace.join("\n") # TODO: amir - Revisit this.
       end
 
+      def render_data(data: {}, status: :ok, include: nil)
+        render json: data, status:, include:, root: 'data'
+      end
+
+      def render_error(errors: [], status: :ok)
+        render json: {
+          errors:
+        }, status:
+      end
+
```

Comment:
Because of how serialization works we'll need to split this method into 2 - eventually we'll get rid of `render_json` entirely